components/apimgt/org.wso2.carbon.apimgt.persistence/src/main/java/org/wso2/carbon/apimgt/persistence/APIConstants.java

```diff
@@ -244,6 +244,7 @@ public static class Monetization {
     public static final String TAG_COLON_SEARCH_TYPE_PREFIX = "tag:";
     public static final String NAME_TYPE_PREFIX = "name";
     private static final String PROVIDER_SEARCH_TYPE_PREFIX = "provider";
+    public static final String API_PROVIDER_SUFFIX_SLASH = "provider/";
```

Comment:
Shall we reuse the PROVIDER_SEARCH_TYPE_PREFIX variable here?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/12200/commits/c4e957a25dc551aac6d2e8d6886d4a753b3a2e82